frontend/pages/index.tsx

```diff
@@ -33,12 +50,59 @@ export default function Home(): React.ReactNode {
           Dialog!
         </Link>
       </Heading>
-      <p>{formatMessage(messages.hello)}</p>
-      <p>{formatMessage(messages.welcomeMessage)}</p>
+      <p>{f("hello")}</p>
+      <p>{f("welcomeMessage")}</p>
       <p>Current locale: {locale}</p>
       <p>Default locale: {defaultLocale}</p>
       <p>Configured locales: {JSON.stringify(locales)}</p>
 
+      <main className={styles.main}>
+        <Box bg="brand.100" w="100%" h="100%" p={10}>
+          <Text fontSize="3xl" align="center" color="white" fontWeight="bold">
+            {f("featuresTitle")}
+          </Text>
+          <br></br>
+          <br></br>
+          <SimpleGrid columns={{ sm: 2, md: 3 }} spacing={14}>
+            <FeatureCard
+              title={f("feature1")}
+              description={f("feature1msg")}
+              icon={BiPalette}
+            />
+
+            <FeatureCard
+              title={f("feature2")}
+              description={f("feature2msg")}
+              icon={ImWrench}
+            />
+
+            <FeatureCard
+              title={f("feature3")}
+              description={f("feature3msg")}
+              icon={FaUniversity}
+            />
+
+            <FeatureCard
+              title={f("feature4")}
+              description={f("feature4msg")}
+              icon={IoAccessibility}
+            />
+
+            <FeatureCard
+              title={f("feature5")}
+              description={f("feature5msg")}
+              icon={RiCodeSSlashLine}
+            />
+
+            <FeatureCard
+              title={f("feature6")}
+              description={f("feature6msg")}
+              icon={RiOpenSourceFill}
+            />
+          </SimpleGrid>
```

Comment:
Great work on the i8n! One suggestion I could make to help clean this up is to add these id's to the messages object above and utilize that instead of using string literals, it would help later on if we choose to implement the translation manager, or if we need to swap out ids!